index.bs

```diff
@@ -4606,7 +4664,8 @@ script=].
       }
 
       script.AddPreloadScriptParameters = {
-        expression: text;
+        arguments: [*script.ArgumentValue],
```

Comment:
Do we maybe want to have `arguments` parameter as optional, the same as in callFunction?